nsxt/policy_common.go

```diff
@@ -111,6 +111,14 @@ func getDataSourceStringSchema(description string) *schema.Schema {
 	}
 }
 
+func getDataSourceRequiredStringSchema(description string) *schema.Schema {
```

Comment:
I don't think there is anything specific to data source here, so suggest renaming (this func is used in resource as well)

Reply:
Make sense. Changed.